apps/ledger-live-desktop/src/newArch/Collectibles/components/DetailDrawer/components/PanAndZoom.tsx

```diff
@@ -0,0 +1,97 @@
+import React, { memo } from "react";
+import { createPortal } from "react-dom";
+import { PanAndZoomProps, PanAndZoomBodyProps } from "LLD/Collectibles/types/DetailDrawer";
+import { Media } from "LLD/Collectibles/components/index";
+import PrismaZoom from "react-prismazoom";
+import IconCross from "~/renderer/icons/Cross";
+import styled from "styled-components";
+
+const Container = styled.div`
+  background-color: rgba(0, 0, 0, 0.4);
```

Comment:
```suggestion
 background-color: ${({ theme }) => rgba(theme.colors.neutral.c100,0.4)};
```
To handle Dark/light theme